env.example.yml

```diff
@@ -1,7 +1,9 @@
 LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as the subnet)
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
+HERE_API_KEY: INSERT HERE API KEY HERE
 GEOCODE_EARTH_API_KEY: INSERT GEOCODE.EARTH API KEY HERE
 GEOCODE_EARTH_URL: https://api.geocode.earth/v1
```

Comment:
We should probably add a note to this one saying that it can be removed if geocode.earth is not in use.